File: .oracle/sonar/projects/oracle-py/src/s6319_good.py
from aws_cdk import aws_sagemaker as sagemaker

notebook = sagemaker.CfnNotebookInstance(
    scope,
    "notebook",
    instance_type="ml.t2.medium",
    role_arn="arn:aws:iam::123456789012:role/notebook",
    kms_key_id="alias/notebook",
)

File: .oracle/sonar/projects/oracle-js/src/s6319_good.js
import { aws_sagemaker } from 'aws-cdk-lib';

// Notebook instance encrypted with a customer-managed KMS key.
new aws_sagemaker.CfnNotebookInstance(this, 'Notebook', {
  kmsKeyId: 'arn:aws:kms:eu-central-1:123456789012:key/1234abcd-12ab-34cd-56ef-1234567890ab',
});
